### DISCUSSION TITLE
Crosschat & Serverchat Tags

### DISCUSSION THREAD

**Discussion by JXYZP-G** [gameserverapp/Platform#381] [Suggestions]
it would be nice to have the possibility to add donator tags to crosschat and serverchat. 

**Support Tiers** 
For example V.I.P tier you have a crosschat variable input `[{servername}] {charname} {groupname}: {message}` would be nice that there will be a {(highest)custom role}

Example:
`[{servername}][tier] {charname} {groupname}: {message}`
[Ragnarok][V.I.P] JXYZP (Tribe of JXYZP): here is my message :) 

Features for chat:

- Color role (Tag color)
- Custom Role Display Name (In-game)(AKA tags)
- Player possibility to hide/show mapname 

There are probably more functionality's but i forgot them if any of you have more ideas that would fit right please reply below!

**Comment by zerafox** [comment]
I would LOVE to be able to have custom coloring or a tag or something for supporter/VIP tiers <3
